guides/common/modules/proc_upgrading-smartproxy-server.adoc

```diff
@@ -44,8 +59,11 @@ endif::[]
 ifdef::foreman-el[]
 # {project-package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm
 endif::[]
+ifdef::foreman-deb[]
+// What's the right thing to add for Debian?
+endif::[]
```

Comment:
@ekohl Here (and quite possibly elsewhere) I'm struggling with what the steps should look like for Debian. Can you take a look?

Reply:
The procedure can be the same but I don't know what the right repository for Debian is. I tried looking for it and failed. Can you help?

Reply:
I did my best to add this, and I also implemented tabs for the Debian procedure while I was at it.